### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by radhak21**
Added storage setting under Esper setting and added a note for tags drop-down.

### CODE REVIEW

**Review by kirananto — DISMISSED** [review]

**Review by ankitgupta29 — APPROVED** [review]

**Review by kzotter — CHANGES_REQUESTED** [review]
Capitalize Bluetooth

**Review by kzotter — APPROVED** [review]

**Review by kzotter — APPROVED** [review]

**Review by kzotter — APPROVED** [review]

**Review by VineethaVR — COMMENTED** [review]
Why are we adding a link to old docs page? It should be relative to the new docs.

"Factory Reset Protection (FRP) places an auto-lock on a GMS-certified device if a user factory resets it with soft or hard keys. FRP will auto-lock the device and keep it locked until a pre-authorized Google account ID is entered. Additional information about getting your Google Account ID can be found in the [here](https://docs.esper.io/home/console/policy-management/getgoogleaccount.html#how-to-get-google-id-and-use-frp-factory-reset-protection-feature)."

**Review by colesper — APPROVED** [review]

File: docs/compliance-policy/frp.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+:::tip
+This "id" is the Google Account Id that you need to enter on the Esper Dashboard under the Factory Reset Protection button on the Policy page.
+:::
+
+![Esper Get Google Account Id](./images/frp/4-googleaccountid.png)
+
+Entering this ID under the Factory Reset Toggle button on Policy (or Template) will authorize the user to unlock the device post factory reset.
+
+As a result, an unauthorized user can not simply factory reset your device and use it against your permission.
```
Comment (colesper) [inline docs/compliance-policy/frp.md:26]: Change "can not" to "cannot"

**Review by colesper — CHANGES_REQUESTED** [review]
Commented in-line. But "can not" should be "cannot".

**Review by VineethaVR — APPROVED** [review]

**Review by colesper — APPROVED** [review]

File: docs/compliance-policy/create-policy.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ This application offers the following settings:
 -   Auto-Rotation: Turn auto-rotation On or Off.
 -   Factory Reset: Perform a factory reset of the device, regardless of the compliance policy applied.
 -   Kiosk App Selection: Change the app that runs in kiosk mode. The device user can choose any installed app on the device to act as the kiosk app, with the Esper Console kept in sync on the configuration stats.
+- Storage: Helps to grant permissions to any directories on the device. On devices running Android 10 or below, it is advised that the admins inform the users to grant permissions to the root folder. On Android 11 devices, permission cannot be granted to the root folders and download folder.
```
Comment (ankitgupta29) [inline docs/compliance-policy/create-policy.md:100]: Console admin informs the device users

Comment (ankitgupta29) [reply docs/compliance-policy/create-policy.md:100]: root folder? Can we specify example what does this mean like /sdcard/ etc?

File: docs/compliance-policy/create-policy.md [hunk 3]
```diff
@@ -107,7 +108,7 @@ This application offers the following settings:
 -   Keyboard: Navigate to Android settings on the Android 8.1 and lower devices to change the keyboard settings.
 -   Accessibility: Navigate to Android settings on the Android 8.1 and lower devices to change the accessibility settings.
 -   Input Selection: Navigate to Android settings on the Android 8.1 and lower devices to change the input selection for the device.
--   Bluetooth: Navigate to Android settings on the Android 8.1 and lower devices to change the bluetooth setting.
+-   Bluetooth: Navigate to Android settings on the Android 8.1 and lower devices to change the Bluetooth setting.
```
Comment (ankitgupta29) [inline docs/compliance-policy/create-policy.md:111]: Navigate to the Android System Settings app

File: docs/compliance-policy/create-policy.md [hunk 4]
```diff
@@ -261,6 +262,10 @@ This feature allows the installation of applications from unknown sources on the
 
 For example, if the device user chooses Chrome, they will be able to download and install applications from the Chrome browser on the device. If turned Off, the device user will not be allowed to install applications from any unknown sources.
 
+### Bluetooth setting via Policy
+If you would like the device user to use bluetooth on the device, set the bluetooth toggle button to On; if you don’t, set it to Off.
```
Comment (ankitgupta29) [inline docs/compliance-policy/create-policy.md:266]: bluetooth ->Bluetooth

File: docs/content-management/transfer.md [hunk 5]
```diff
@@ -19,6 +19,9 @@ Step 3:  Provide the destination file path on the devices. One example of an int
 Once complete, click **Transfer**. 
 
 You can transfer files to internal storage or removable storage device like SD cards. All the removable storage devices will have a unique eight-character ID. You just need to add a prefix "ESPER_EXTERNAL" if the ID is not known.
+
+You can append the path with the "forced/" keyword ("forced/storage/emulated/0/Esper/NewFolder/"), which will ask users permission for that directory (if the directory exists). On granting access, the files will be saved anywhere in internal storage, public directories, and removable storage.
```
Comment (ankitgupta29) [inline docs/content-management/transfer.md:23]: add example

You can append the path with the "forced/" keyword (example: "forced/storage/emulated/0/Esper/NewFolder/")

**Review by VineethaVR — APPROVED** [review]

**Pull request merged**